tools/submit-pull-request

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+#
+# Copyright European Organization for Nuclear Research (CERN)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Martin Barisits, <martin.barisits@cern.ch>, 2017
+
+
+import commands
+import sys
+
+
+def submit_pull_request(branch, message, label):
+    op = commands.getstatusoutput('hub pull-request -m "%s" -b %s -l %s' % (message, branch, label))
```

Comment:
label not supported with the default version (not pre release)
I think we can drop it ? 
in the future the link to the issue can be also added there: at the endhttps://github.com/rucio/rucio/issues/<number>

if the association pull request <-> issue is missing

Reply:
-label is supported in hub since 2016, I think you are just using a very old version.
The linking of pull-requests to issues is a strongly discouraged behaviour, even so much that it is deprecated in the API, that's why I took it out.

Reply:
well it's the default and latest version in homebrew. the pre releases are maybe not considered final by them there. Why are the possible labels here ?

Reply:
I would like to have it as it is immediately clear if it is a patch/feature/hotfix. But I see that it is a bit difficult to require the binary. For CC7 you need the binary anyway, as the default version does not work. Alternatively I can try to make the script just directly with the REST commands, but that will require some more work.

Reply:
well to do pull request on github, you need need hub installed. Then to have with git I just have the alias:
alias git='hub'
then
git pull-request -m 'readme' ...